Mathlib/Init/Function.lean

```diff
@@ -94,16 +94,16 @@ theorem id_comp (f : α → β) : id ∘ f = f := rfl
 #align function.left_id Function.id_comp
 #align function.comp.left_id Function.id_comp
 
-@[deprecated] alias left_id := id_comp -- Deprecated since 14 January 2014
-@[deprecated] alias comp.left_id := id_comp -- Deprecated since 14 January 2014
+@[deprecated] alias left_id := id_comp -- 2024-01-14
```

Comment:
Why not use the `since` syntax here and below?

Reply:
Good idea, done.